scripts/build-info.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Concurrent Technologies Corporation 2023
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import json
+import subprocess
+import argparse
+from os import path
+
+parse = argparse.ArgumentParser()
+parse.add_argument("path")
+args = parse.parse_args()
+
+
+os_info = subprocess.getoutput(["uname -nr"])
+git_info = subprocess.getoutput(["git log -n 1"])
+time_info = subprocess.getoutput("date")
+
+
+if path.isfile(args.path):
+    try:
+        f = open(args.path, "r")
+        data = json.load(f)
+        f.close()
+        if "OS_UNKNOWN" in data["os_info"]:
```

Comment:
```python
if "OS_UNKNOWN" in data["os_info"]:
```

Since this is the script that will be run in dev env or CI, I dont think you need to check and see if a value has already been set.  Should be safe to blindly overwrite with the results of the subprocess calls.